### PR TITLE
Update Feature Management release 2.6.1

### PR DESCRIPTION
**Pull request by zhiyuanliang-ms**

### CODE REVIEW

File: releaseNotes/Microsoft.Featuremanagement.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ If all of three filters are registered:
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeB, FilterB will be used and if context's type is TypeC, FilterC will be used.
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeF, FilterA will be used.
 
+## 2.6.1 - June 28, 2023
+
+### Bug fix
+
+* Adds edge case for EvaluateAsync call that doesn't use context from FeatureManager.
```
Comment (zhenlan) [inline releaseNotes/Microsoft.Featuremanagement.md:91]: Fixed edge cases for ...

Is there an issue we can reference here?

Comment (zhiyuanliang-ms) [reply releaseNotes/Microsoft.Featuremanagement.md:91]: https://github.com/microsoft/FeatureManagement-Dotnet/issues/244

Comment (zhiyuanliang-ms) [reply releaseNotes/Microsoft.Featuremanagement.md]: Reference added.

File: releaseNotes/Microsoft.Featuremanagement.md [hunk 2]
```diff
@@ -84,6 +84,12 @@ If all of three filters are registered:
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeB, FilterB will be used and if context's type is TypeC, FilterC will be used.
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeF, FilterA will be used.
 
+## 2.6.1 - June 28, 2023
+
+### Bug fix
+
+* Adds [edge case](https://github.com/microsoft/FeatureManagement-Dotnet/issues/244) for EvaluateAsync call that doesn't use context from FeatureManager.
```
Comment (zhenlan) [inline releaseNotes/Microsoft.Featuremanagement.md:91]: I understand we were not consistent, but we should
- use past tense
- put the referenced issue in the end

So suggest:

Fixed an edge case for ... [#244](https://github.com/microsoft/FeatureManagement-Dotnet/issues/244)

**Review by zhenlan — APPROVED** [review]

**Pull request merged**